### PR TITLE
Shared repository indicator

### PR DESCRIPTION
**Pull request by piotr-travisci**
https://travisci.assembla.com/spaces/Web/tickets/realtime_cardwall?ticket=278

### CODE REVIEW

File: app/templates/components/my-build.hbs [hunk 1]
```diff
@@ -26,7 +26,7 @@
           @models={{array this.build.repo.provider this.build.repo.ownerName this.build.repo.vcsName}}
           class="inner-underline"
         >
-          {{this.build.repo.name}}
+          {{this.build.repo.name}}{{#if this.build.repo.shared}} <SvgImage @name="icon-shared" @alt="Shared repository" @class="icon shared" />{{/if}}
```
Comment (artursmirnov) [inline app/templates/components/my-build.hbs]: Here too:
```suggestion
          {{this.build.repo.name}}
          {{#if this.build.repo.shared}}
            <SvgImage @name="icon-shared" @alt="Shared repository" @class="icon shared" />
          {{/if}}
```

File: app/templates/components/owner-repo-tile.hbs [hunk 2]
```diff
@@ -20,7 +20,7 @@
           class="label-align"
         >
           <span class="repo-title-text">
-            {{this.repo.name}}
+            {{this.repo.name}}{{#if this.repo.shared}} <SvgImage @name="icon-shared" @alt="Shared repository" @class="icon shared" />{{/if}}
```
Comment (artursmirnov) [inline app/templates/components/owner-repo-tile.hbs]: And here:
```suggestion
            {{this.repo.name}}
            {{#if this.repo.shared}}
              <SvgImage @name="icon-shared" @alt="Shared repository" @class="icon shared" />
            {{/if}}
```

File: app/templates/components/repos-list-item.hbs [hunk 3]
```diff
@@ -4,7 +4,7 @@
       <LinkTo @route="repo" @models={{array this.repo.provider this.repo.ownerName this.repo.vcsName}}>
         <StatusIcon @status={{this.repo.currentBuild.state}} />
         <span data-test-repos-list-item-slug class="label-align inner-underline">
-          {{this.repo.slug}}
+          {{this.repo.slug}}{{#if this.repo.shared}} <SvgImage @name="icon-shared" @alt="Shared repository" @class="icon shared" />{{/if}}
```
Comment (artursmirnov) [inline app/templates/components/repos-list-item.hbs]: And here:
```suggestion
          {{this.repo.slug}}
          {{#if this.repo.shared}}
            <SvgImage @name="icon-shared" @alt="Shared repository" @class="icon shared" />
          {{/if}}
```

**Comment by artursmirnov** [comment]
@piotr-travisci thank you for addressing the feedback! It looks good to me visually as well, the only thing that caught my eye is this:

image

When you hover on either image, it just shows you a common tooltip for both icons. Would it be better if we show separate tooltips for icons to clarify their meaning, e.g. "Shared repository", "Private repository"? Also I'm a bit uncertain about the term "Shared" repository - it doesn't seem like a common term, and I wouldn't understand it's meaning without knowing the task. Is it something that we developed ourselves?

**Comment by piotr-travisci** [comment]
Thank you @artursmirnov ! I'm glad you catch this.
I refactored tooltip and now it shows proper tooltip to each icon.
"Shared repository" is a term provided by our Creative team.

**Pull request merged**